docs/dbg.md

```diff
@@ -0,0 +1,162 @@
+# DBG helper
+
+
+### store
+
+`dbg.store` is a convenient way for getting the redux store.
+
+```js
+const store = dbg.store.getState()
+
+// select a source
+const source = dbg.store.getState().sources.sources.first();
+```
+
+### actions
+
+`dbg.actions` is a convenient way for getting the redux actions.
+The actions are bound so you can call them directly!
+
+```js
+const source = dbg.store.getState().sources.sources.first();
+dbg.actions.selectSource(source.get("id")))
+```
+
+### selectors
+
+`dbg.selectors` is a convenient way for getting the redux selectors.
```

Comment:
`is a convenient way` is used quite often, perhaps this sentence fragment is not needed in the above three cases, and we can keep this shorter -- that way there is less to read :)